src/routes/index.js

```diff
@@ -1,6 +1,20 @@
 module.exports = function(express) {
+  var express = require('express');
+  var histories = require('../models/histories.js');
+  var db = require('../models/db.js');
   var router = express.Router();
 
+
+// NOTES___________________________________________________
```

Comment:
This does not match the format for commenting as defined in the airbnb style guide. Also, distill your comments down to a shorter and more cohesive statement. Please refine.